### PR TITLE
Let users select provider OS

### PR DESCRIPTION
**Pull request by iamemilio**
Enables users to select which provider os they want to use in the generate-yaml.sh script
Referencing: #88

Special Notes:
:)

### CODE REVIEW

File: cmd/clusterctl/examples/openstack/generate-yaml.sh [hunk 1]
```diff
@@ -12,8 +12,14 @@ print_help()
   echo "-h, --help                show brief help"
   echo "-f, --force-overwrite     if file to be generated already exists, force script to overwrite it"
   echo "-c, --clouds [File]       specifies an existing clouds.yaml file to use rather than generating one interactively"
+  echo "--provider-os [os name]   select the operating system of your provider environment"
+  echo "                            Supported Operating Systems: ubuntu, centos"
```
Comment (hidekazuna) [inline cmd/clusterctl/examples/openstack/generate-yaml.sh]: This script ended with error without --provider-os option. Please add "Mandatory"

File: cmd/clusterctl/examples/openstack/generate-yaml.sh [hunk 2]
```diff
@@ -12,8 +12,14 @@ print_help()
...
+  echo ""
 }
 
+# Supported Operating Systems
+declare -a arr=("centos" "ubuntu")
+
 SCRIPT=$(basename $0)
 while test $# -gt 0; do
```
Comment (hidekazuna) [inline cmd/clusterctl/examples/openstack/generate-yaml.sh]: --provider-os is mandatory. Please take care of it.

**Comment by iamemilio** [comment]
@hidekazuna the issue has been resolved. Thanks for the review!

File: cmd/clusterctl/examples/openstack/generate-yaml.sh (hunk 2 above)
Comment (hidekazuna) [reply cmd/clusterctl/examples/openstack/generate-yaml.sh]: Thanks for updating. It works fine now.

**Comment by iamemilio** [comment]
/assign @dims 

It was the robots idea: "We suggest the following additional approver: dims"

**Pull request merged**